Tasks/Vadzim/Task3.1/query06.sql

```diff
@@ -0,0 +1,10 @@
+﻿--Query 6
+SELECT TOP(10)
+    ssod.SalesOrderID,
+    MAX(ssod.UnitPrice) AS TotalPrice
```

Comment:
there is more that 1 record per sale order id so summary of all values of sale order id should be calculated
also Total price would be not just UnitPrice but would need to have qnt & discount get total (LineTotal is a calc column that have done it to us :) )